next.config.js

```diff
@@ -1,4 +1,15 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+    images: {
```

Comment:
Can you explain this to me please?

Reply:
this is how next js accept the src attribute of img tag which get the image from another origin.